projects/systemd/project.yaml

```diff
@@ -7,3 +7,7 @@ sanitizers:
 auto_ccs:
   - jonathan@titanous.com
   - zbyszek@in.waw.pl
+  - dimitri.ledkov@canonical.com
```

Comment:
Monorail started sending spurious notifications 10 days ago and looking at "CC: a_deleted_user" at https://bugs.chromium.org/p/oss-fuzz/issues/detail?id=39238#c5 my guess would be that it has something to do this email address not being active anymore. Judging by https://github.com/google/oss-fuzz/issues/6825 the only way to stop those notifications is to remove or replace it. @xnox since it's your email address could you take a look?

Reply:
I opened https://github.com/google/oss-fuzz/pull/11702 where I dropped that address. It should prevent monorail from sending notifications every other day. It should be possible to open a separate PR and add another address if necessary.